decidim-dev/lib/decidim/dev/test/w3c_rspec_validators_overrides.rb

```diff
@@ -1,5 +1,41 @@
 # frozen_string_literal: true
 
+# This is a temporary fix to ignore some HTML/CSS validation issues with the
+# Decidim HTML validation process.
+#
+# See: https://github.com/decidim/decidim/pull/10014
+# Related: 
```

Comment:
Uops, the dangers of not using a real IDE and doing it through GH UI: I've left a trailing whitespace. Can you remove it please? Thanks
```suggestion
# Related:
```